modules/vmseries/README.md

```diff
@@ -80,6 +81,7 @@ No modules.
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | <a name="input_accelerated_networking"></a> [accelerated\_networking](#input\_accelerated\_networking) | Enable Azure accelerated networking (SR-IOV) for all network interfaces except the primary one (it is the PAN-OS management interface, which [does not support](https://docs.paloaltonetworks.com/pan-os/9-0/pan-os-new-features/virtualization-features/support-for-azure-accelerated-networking-sriov) acceleration). | `bool` | `true` | no |
+| <a name="input_app_insights_settings"></a> [app\_insights\_settings](#input\_app\_insights\_settings) | Application Insights variable list:<br>- `create`: (optional\|bool) Enable usage of the Application Insights. The default vailue is set to `false`.<br>- `name`: (optional\|string) The name of the Applications Insights instance. Can be `null`, in which case a default name is auto-generated.<br>- `workspace_mode`: (optional\|bool) If `true` the Application Insights mode is set to \"Workspace-based\". Otherwise, the legacy \"Classic\" mode is used.<br>- `metrics_retention_in_days`: (optional\|number) Specifies the retention period in days. Possible values are 0, 30, 60, 90, 120, 180, 270, 365, 550 or 730. Defaults to 90.<br>- `application_type`: (optional\|string) Specifies the type of Application Insights to create. Valid value is `other`.<br>- `log_analytics_name`: (optional\|string) The name of the Log Analytics workspace. Can be `null`, in which case a default name is auto-generated.<br>- `log_analytics_sku`: (optional\|string) Azure Log Analytics Workspace mode SKU. The default value is set to \"PerGB2018\".<br><br>Example:<pre>{<br>      create                    = true<br>      name                      = "AppInsights"<br>      workspace_mode            = true<br>      metrics_retention_in_days = 30<br>      application_type          = "other"<br>      log_analytics_name        = "LogAnalyticsName"<br>      log_analytics_sku         = "PerGB2018"<br>  }</pre> | `map(any)` | `{}` | no |
```

Comment:
`metrics_retention_in_days` default is set to null, not 90 
`  retention_in_days   = try(var.app_insights_settings.metrics_retention_in_days, null)`

Reply:
True, the 90 is the azure default value if you set variable to `null`. But, I will change this description.